# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt
--- a/third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt	Fri Jan 27 20:35:34 2023 +0000
+++ b/third_party/rust/mp4parse/link-u-avif-sample-images/star.input.txt	Fri Jan 27 22:54:08 2023 +0200
@@ -1,9 +1,9 @@
-file 'star.png'
-duration 0.1
-file 'star90.png'
-duration 0.1
-file 'star180.png'
-duration 0.1
-file 'star270.png'
-duration 0.1
-file 'star.png'
+file 'star.png'
+duration 0.1
+file 'star90.png'
+duration 0.1
+file 'star180.png'
+duration 0.1
+file 'star270.png'
+duration 0.1
+file 'star.png'